src/TroykaOLED.h

```diff
@@ -160,6 +159,7 @@ class TroykaOLED {
     void print(int32_t n, int x = OLED_THIS, int y = OLED_THIS, uint8_t base = DEC);
     void print(uint32_t n, int x = OLED_THIS, int y = OLED_THIS, uint8_t base = DEC);
     void print(double n, int x = OLED_THIS, int y = OLED_THIS, uint8_t digits = 2);
+	void print(char ch, int x, int y);
```

Comment:
Why moved here? Why with wrong indentation. Suggest putting it back where it was before the latest commit.

Reply:
I didn't notice that notepad++ didn't replace tab with 4 spaces.